src/api/routes/v1/info.ts

```diff
@@ -82,6 +82,17 @@ const info = {
       example:
         'https://data.xrpl.org/v1/network/amendments/vote/{network}/{identifier}',
     },
+    {
```

Comment:
Since this is for internal use I don't think we need to expose it in info

Reply:
I meant we don't show it in the info when users hit https://data.xrpl.org, but still keep it within architecture document